### DISCUSSION TITLE
converting grayscale sRGB tif  to  grayscale sRGB jpg

### DISCUSSION THREAD

**Discussion by DanielDrs** [ImageMagick/ImageMagick#5635] [Digital Image Processing] [answered]
Hallo,

I use     convert a.tif b.jpg   in a large tcsh-script whereas a.tif  is 16-bit sRGB tif.   Usually, I get a  8-bit sRGB jpg-image b.jpg.

In some rare cases, the red, green, and blue-values of a.tif are identical, i.e., a.tif looks like a gray-scale image, but it is still a 16-bit sRGB tif.   In these rare cases,  convert a.tif b.jpg  produces an 8-bit Gray 256c jpg-image. 

However,  I want to have a 8-bit sRGB jpg-image b.jpg.

Best regards, Daniel

**Comment by snibgo** [comment, marked as answer]
Try "`-type truecolor`" before writing the output JPG.

**Reply by DanielDrs** [reply]
Thanks!!!
